docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Ubuntu/README.md

```diff
@@ -1 +1,182 @@
-Ubuntu specific instructions
\ No newline at end of file
+# Installing LORIS In-Depth
+
+This guide assumes an installation on an Ubuntu server. 
+
+If you are using CentOS, please visit the CentOS installation guide for 
+instructions on installation.
+
+We also have instructions for how to set up LORIS for macOS. However, only
+ Ubuntu and CentOS are officially supported by the team so we may be unable to
+ help you with issues arising on other systems.
+
+
+When you've completed this guide, further set up is required. Please visit
+[the Setup page](https://github.com/aces/Loris/wiki/Setup) in order to be
+able to use LORIS properly.
+
+## Prequisities
+
+### Server Stack
+
+LORIS requires a LAMP stack in order to run, specifically:
+* Apache 2.4
+* MySQL 5.7 (or MariaDB 10.1) (or higher)
+* PHP 7.3 (or higher)
+
+Additionally, LORIS requires the following package managers:
+* npm 8.0 (or higher)
+* composer
+
+This guide does not cover installation of these requirements.
+
+### Apt Packages
+The following Ubuntu packages are required and should be installed using 
+`sudo apt install ...`.
+
+* git
+* curl
+* wget
+* zip
+* unzip
+* php-json
```

Comment:
php-json and php7.3-json?

Reply:
Thanks I deleted `php73-json`.

Reply:
shouldn't it be the opposite? keep php73-json.

Reply:
I tried running `php-json` on my VM and it started downloading php74-json on its own. i think it's a smart(er) parent package that takes care of the rest.